src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/ProteinsList.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+
+import ProteinsListItem from './proteins-list-item/ProteinsListItem';
+
+import { fetchGene } from '../../../shared/rest/rest-data-fetchers/geneData';
+
+import { Gene } from '../../../types/gene';
+
+import styles from './ProteinsList.scss';
+
+type ProteinsListProps = {
+  geneId: string;
+};
+
+type ProteinListWithDataProps = {
+  gene: Gene | null;
+};
+
+const ProteinsList = (props: ProteinsListProps) => {
+  const [geneData, setGeneData] = useState<Gene | null>(null);
+
+  useEffect(() => {
+    fetchGene(props.geneId).then((result) => {
+      if (result) {
+        setGeneData(result);
+      }
+    });
+  }, [props.geneId]);
+
+  return <ProteinListWithData gene={geneData} />;
+};
+
+const ProteinListWithData = (props: ProteinListWithDataProps) => {
```

Comment:
Note how the number of the word Protein is inconsistent throughout this PR (ProteinList vs ProteinsList; not just in this file). Pick one of the forms.

Reply:
Ah definitely has to be ProteinsList. I'll change it to that.

Reply:
✔️